project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
hrm.  seems like jetty should be forcing it to come in as a transitive dep, and if it's not, we should (ultimately) probably fix that in jetty... but... I don't know if it's worth fixing right this second.

ugh.  i think this means that anyone who is using jetty testutils is going to have to deal with this... so yeah i guess maybe i do think we should fix it in jetty now... wdyt?